pkgs/tools/misc/keep-presence/default.nix

```diff
@@ -0,0 +1,47 @@
+{ nixpkgs ? import <nixpkgs> {}, pythonPkgs ? nixpkgs.pkgs.python3Packages }:
```

Comment:
Nixpkgs is never imported from within nixpkgs, just make this file a function that takes its inputs as arguments and `callPackage` will do the rest.